react/postalCodeAutoCompleteAddress.test.js

```diff
@@ -76,4 +76,26 @@ describe('postalCodeAutoCompleteAddress()', () => {
       callback,
     })
   })
+
+  it.only('should keep address id when auto completing fields', done => {
```

Comment:
You forgot to remove the `only` 👀
```suggestion
  it('should keep address id when auto completing fields', done => {
```

Reply:
NICE!